applications/machine_learning/README.rst

```diff
@@ -197,8 +197,8 @@ See :ref:`zephyr:thingy52_nrf52832` documentation for details.
 Programming Thingy:53
 =====================
 
-The Thingy:53 does not have the J-Link debug IC and the application uses MCUboot bootloader.
-See :ref:`ug_thingy53` for detailed description about working with Thingy:53 and information about how to program it.
+If the application is used with a Thingy:53, it complies with Thingy:53 application guide and enables additional features.
```

Comment:
This seems good. It should now become the text for the shortcut, and the shortcut used in all places where this text is currently.

Reply:
Again, to me it seems more correct to replace "used with a Thingy:53" with "built for the Thingy:53".

Reply:
Ok, I will update